include/deal.II/base/vectorization.h

```diff
@@ -2707,6 +2707,25 @@ vectorized_transpose_and_store(const bool                    add_into,
 
 
 /**
+ * Relational operator == for VectorizedArray
+ *
+ * @relates VectorizedArray
+ */
+template <typename Number>
+inline DEAL_II_ALWAYS_INLINE
+bool
+operator == (const VectorizedArray<Number> &lhs,
```

Comment:
The error message uses alternately `VectorizedArray` and `dealii::VectorizedArray`. You may want to look into why that is so -- I bet that's the root cause for the problem.